### PR TITLE
[WIP] Remove klog as a dependency

### PR DESCRIPTION
**Pull request by palash25**
Fixes: https://github.com/kubernetes/kube-openapi/issues/117

### CODE REVIEW

**Comment by sttts** [comment]
klog is a transitive dependency from gengo. We get it anyway.

**Comment by palash25** [comment]
> klog is a transitive dependency from gengo. We get it anyway.

correct me if I am wrong but I think that the issue referenced is about eliminating the use of logging in kube-openapi and having that dependency due to some other dep would be irrelevant to this PR. Am I right?

File: cmd/openapi-gen/openapi-gen.go [hunk 1]
```diff
@@ -28,12 +28,9 @@ import (
 	"k8s.io/kube-openapi/pkg/generators"
 
 	"github.com/spf13/pflag"
-
-	"k8s.io/klog"
```
[the remark this answers is not in the document]
Comment (palash25) [reply cmd/openapi-gen/openapi-gen.go]: removed this change :+1:

**Comment by sttts** [comment]
@palash25 all the remaining changes are also about the openapi-gen command, aren't they? The same applies as in https://github.com/kubernetes/kube-openapi/pull/134#pullrequestreview-199515734

**Pull request closed without merging**